gslib/__main__.py

```diff
@@ -408,6 +398,9 @@ def main():
     else:
       command_name = args[0]
 
+    # Now that we have the command name, update the Boto user agent.
+    boto.UserAgent += ' command/%s' % command_name
```

Comment:
Is the command_name getting repeated?. We are already adding this info via `GetUserAgent`?

Reply:
Sorry I think you and I caught this around the same time.